Makefile

```diff
@@ -162,7 +162,6 @@ docker-build: ## Build docker image with the manager.
 		--tag kubeshop/kusk-gateway:$(shell git describe --tags $(shell git rev-list --tags --max-count=1)) \
 		--file ./build/manager/Dockerfile \
 		.
-	minikube image --profile kgw load kubeshop/kusk-gateway:$(shell git describe --tags $(shell git rev-list --tags --max-count=1))
```

Comment:
We should keep this, I think. Its quicker to build on the local machine and then load it into minikube rather than building directly in minikube

Reply:
You are right. 
Reverted  it